### ISSUE TITLE
[BUG] SVG logos not appear in list of choices

### ISSUE THREAD

**Issue by tuudik** [PhlexPlexico/get5-web#163]
**Describe the bug**

**Which Build**
master

**To Reproduce**
uploaded SVG's to get5-web/get5/static/resource/csgo/materials/panorama/images/tournaments/teams
logos not appearing in list

**Expected behavior**
after uploading svg's the list of those names should appear in list of logo to choose

**Comment by PhlexPlexico** [comment]
If you restart the application does it appear in the list (i.e. restart apache2)?

**Comment by tuudik** [comment]
I did the change like this:
https://github.com/PhlexPlexico/get5-web/compare/master...tuudik:patch-1

And this fixed the issue for me:)

**Comment by PhlexPlexico** [comment]
Perfect. If you wouldn't mind, could you branch off of development and create a PR for that? Turns out I'm bad at indentation. Thanks!

**Comment by PhlexPlexico** [comment]
Going to close this now as a patch was made and it seems to be working.

**Comment by tuudik** [comment]
Btw, have you managed to get panoramas working also in the game? the server tries to fetch png's.

**Comment by MikkelDK** [comment]
@tuudik - PNG doesn't work for "scoreboard display" SVG does.

Do you had try set sv_downloadurl "yourlink.tld/static/resource/csgo/"

**Comment by tuudik** [comment]
@MikkelDK My FastDL server is setup correctly afaik:
"sv_downloadurl" = "http://masin.csgo-server.com"

and SVG are available from here:
http://masin.csgo-server.com/materials/panorama/images/tournaments/teams/

**Comment by MikkelDK** [comment]
@tuudik get5-panel from phlex has own for logos ;) 

Try to do this on your console: sv_allowupload 1 and retry.
There is a plugin, where it can "force downloading .svg" **WITHOUT** sv_allowupload need to be set!

**Comment by tuudik** [comment]
@MikkelDK I think the plugin you are talking about is https://steamcommunity.com/linkfilter/?url=https://github.com/uspeek/CSGO-Team-Logo-Manager/releases/tag/1.4.2panorama ?
I did install that. 

I also tried the sv_allowupload option, but still no result

**Comment by tuudik** [comment]
What get5-panel are you exactly writing about? couldnt find it.

**Comment by PhlexPlexico** [comment]
I've found that both setting that, and the plugin caused the server connect to hang indefinitely. What I just did (since the logos only mattered to me on a stream PC) so I placed the files in my local PC by myself. 

And logos are uploaded from this webpanel to the game server, via get5. It's functionality that's in the app right now.

**Comment by MikkelDK** [comment]
> What get5-panel are you exactly writing about? couldn't find it.

You are using Phlex's get5-web right? it has own "folder/little www" for logos. ;)
Example my looks: http://get5.lindhost.eu/static/resource/csgo/

Normally a "custom plugin" is needed. because, you can't get "clients/players" to set sv_allowupload 1 (it's disabled per client by VALVE) but there is a plugin, where it will "force download". - Example: https://forums.alliedmods.net/showpost.php?p=2482368&postcount=1

**Comment by tuudik** [comment]
The logos in get5-web works perfectly, Phlex version. I also use the Phlex get5_apistats.
I am interested that logos are also shown for the players during game.

So let me rephrase it again:D
CSGO client will try to fetch for PNG's not SVG's.

**Comment by PhlexPlexico** [comment]
It should try to fetch both SVGs and PNGs from the server, if they don't exist on the client machine. Not sure why it would be failing on that aspect
